src/Resources/contao/themes/flexible/icons/shield.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="14" height="14" viewBox="0 0 24 24"><path d="M12 22s8-4 8-10V5l-8-3-8 3v7c0 6 8 10 8 10z" fill="none" stroke="#91979c" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="14" height="14" viewBox="0 0 24 24"><path d="M12 22s8-4 8-10V5l-8-3-8 3v7c0 6 8 10 8 10z" fill="none" stroke="#fff" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/></svg>
```

Comment:
Good catch! 👍

Reply:
The fix is wrong though. The stroke color needs to be `#91979c` in the regular menu and `#fff` in the mobile menu, therefore we need two separate icons (similar to `profile.svg` and `profile_dark.svg`).

Reply:
There're already two separate icons. See the dark version of the shield icon: https://github.com/contao/core-bundle/edit/master/src/Resources/contao/themes/flexible/icons/shield_dark.svg

All other dark icons also have the color `#222`:
* `exit_dark.svg`: https://github.com/contao/core-bundle/edit/4.6/src/Resources/contao/themes/flexible/icons/exit_dark.svg
* `profile_dark.svg`: https://github.com/contao/core-bundle/edit/4.6/src/Resources/contao/themes/flexible/icons/profile_dark.svg

Reply:
You are right, sorry.